src/pages/explore/ui/pairs.tsx

```diff
@@ -51,8 +54,49 @@ export const ExplorePairs = () => {
 
   const [search, setSearch] = useState('');
 
+  // Fetch trading pair summaries.
   const { data, isLoading, isRefetching, isFetchingNextPage, fetchNextPage } = useSummaries(search);
 
+  // Create temporary translator to normalize volumes/liquidity to USDC values.
+  // Deep clone is neccesary since objects are passed by reference in typeScript.
+  let translator = data?.pages ? structuredClone(data.pages) : undefined;
+
+  // Extract quote assets from all trading pairs.
+  let quoteAssetsList: string[] = [];
+  if (translator) {
+    quoteAssetsList = collectQuoteAssets(translator);
+  }
+
+  // Get USDC exchange rates for all quote assets.
+  const usdcPriceQueries = useMultipleSummaries(quoteAssetsList);
+
+  // Calculate USDC-denominated volumes and liquidity for each trading pair.
+  // For each pair, scale the values using USDC price of the quote asset.
+  let volumePairs: bigint[] = [];
+  let liquidityPairs: bigint[] = [];
+
+  data?.pages[0]?.forEach((item, index) => {
+    let usdcPrice = usdcPriceQueries[index]?.data as SummaryData;
+
+    if (usdcPrice && 'directVolume' in usdcPrice && 'liquidity' in usdcPrice) {
+      calculateScaledAmount(item, usdcPrice, 'directVolume', volumePairs, index);
+      calculateScaledAmount(item, usdcPrice, 'liquidity', liquidityPairs, index);
+    }
+  });
+
+  // Update volume and liquidity amounts with their respective USDC-normalized values,
+  // skipping USDC pairs.
+  if (translator) {
+    translator.forEach(page => {
+      page.forEach((item, itemIndex) => {
+        if (quoteAssetsList[itemIndex] !== 'USDC') {
+          item.directVolume.valueView.value!.amount!.lo = volumePairs[itemIndex]!;
+          item.liquidity.valueView.value!.amount!.lo = liquidityPairs[itemIndex]!;
```

Comment:
this assignment is what updates the volume and liquidity values to be displayed on the explore page.I think the page should complete the price conversion and asset loading before rendering. Otherwise, the page may load first, and the amounts will update afterward. That's not currently the case, and should be updated to reflect that accordingly.

https://github.com/user-attachments/assets/6397f5ff-079c-40b4-a1ed-09f428edbcdf